src/components/ProductDetail.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Stars from "./Stars";
+
+function ProductDetail(props){
```

Comment:
Make sure to double check the specs. ProductDetail is suppose to represent one product and then App is suppose to do the mapping